testsuite/integration/ws/src/test/java/org/jboss/as/test/integration/ws/wsrm/ReliableCheckHandler.java

```diff
@@ -63,50 +69,52 @@ private synchronized void handleContext(SOAPMessageContext smc) {
         Boolean outboundProperty = (Boolean) smc.get(MessageContext.MESSAGE_OUTBOUND_PROPERTY);
         SOAPMessage message = smc.getMessage();
 
-        invoked++;
-
         if (outboundProperty.booleanValue()) {
-            log.debug("Outgoing message:");
+            log.info("Outgoing message:");
         } else {
-            log.debug("Incoming message:");
+            log.info("Incoming message:");
         }
 
         log.debug("-----------");
         try {
-            JBossLoggingOutputStream os = new JBossLoggingOutputStream(log, Level.DEBUG);
+            JBossLoggingOutputStream os = new JBossLoggingOutputStream(log, Level.INFO);
             message.writeTo(os);
             os.flush();
-            log.debug("");
+            log.info("");
         } catch (Exception e) {
-            log.debug("Exception in handler: " + e);
+            log.info("Exception in handler: " + e);
         }
-        log.debug("-----------");
+        log.info("-----------");
```

Comment:
similary here, we should probably just keep debug, info was needed for debugging